src/components/sidebar/Navigation.tsx

```diff
@@ -0,0 +1,34 @@
+import { HousePlus } from 'lucide-react';
+import Link from 'next/link';
+import React from 'react';
+
+const SideBarMenu = () => {
+  return (
+    <div>
+      <ul className="flex flex-col gap-6 px-6 py-6">
+        <li className="flex items-center gap-4">
+          <HousePlus />
+          <Link href="/trips">Navigation</Link>
```

Comment:
component madu 
repeat ide

and text change madu 
I'll send you icons and page titles